managers/tests/test_views.py

```diff
@@ -258,119 +258,85 @@ def test_delete_project_function_view_should_not_delete_project_on_non_admin_req
         self.assertEqual(Project.objects.all().count(), 1)
 
 
-class ManageTaskActivitiesInProjectViewTests(TestCase):
+class ManageTaskActivitiesTests(TestCase):
```

Comment:
Once you change `ManageTaskActivites` to `ManageTaskActivitesView` please also remember to update test class name

Reply:
changed